mtcpclient/reporting.go

```diff
@@ -18,6 +18,7 @@ func ReportConnectionsStatus(connectionDescriptions []tcpclient.Connection, inte
 	for {
 		fmt.Println(tcpclient.PrintGroupOfConnections(connectionDescriptions))
 		if intervalBetweenUpdates == 0 {
+			fmt.Println(tcpclient.PrintFinalMetricsReport(connectionDescriptions))
```

Comment:
I'd not plug this here... it can be actually be called by printClosureReport, as its part of the Closure report... imho, ReportConnectionStatus differs in objective from this output

Reply:
ok, I will have a look